cli/ceph/orch/orch.py

```diff
@@ -68,3 +73,40 @@ def rm(self, service_name):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def ps(self, **kw):
+        """
+        List daemon running in the node
+        Args:
+            kw (dict): Key/value pairs that needs to be provided to the installer
+
+            Supported keys:
+                hostname (str) : host name
+                service_name (str) : list daemons
+                daemon_type (str) : type of daemon (mon, osd, mgr, mds, rgw).
+                format (str): the type to be formatted(yaml).
+                refresh (bool): whether to refresh or not.
+        """
+        cmd = f"{self.base_cmd} ps {build_cmd_from_args(**kw)}"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def host_ls(self, **kw):
```

Comment:
This method should not be here. Add it in host.py

Reply:
Thanks @vamahaja 
Updated